tests/ifs/ifs.c

```diff
@@ -277,6 +279,7 @@ static int scan_run(struct test *test, int cpu)
         int scan_ret = scan_run_helper(test, i);
         if (scan_ret == IFS_EXIT_CANNOT_START)
         {
+            log_skip(CpuNotSupportedSkipCategory, "Skipping. Not supported on core %d", i);
```

Comment:
You can remove the "Skipping", as it's implied. And we don't need to print the thread number because it comes from the logging header, so just "Not supported on this CPU".